src/main/java/TaskList.java

```diff
@@ -0,0 +1,82 @@
+import java.io.Serializable;
+import java.util.ArrayList;
+
+/**
+ * stores the tasks in a list
+ */
+public class TaskList implements Serializable {
```

Comment:
Not related to the coding standard but I just wanted to say that implementing the Serializable interface was an interesting decision! I had never known about this until now, so thanks for introducing me this 😄